lib/src/dom/selection_api.dart

```diff
@@ -106,9 +117,12 @@ extension type Selection._(JSObject _) implements JSObject {
     int offset,
   ]);
 
-  /// The **`setBaseAndExtent()`** method of the
-  /// [Selection] interface sets the selection to be a range including all or
-  /// parts of two specified DOM nodes, and any content located between them.
+  /// The **`setBaseAndExtent()`** method of the [Selection] interface sets the
+  /// selection to be a range including all or parts of two specified DOM nodes,
+  /// and any content located between them.
+  ///
+  /// The anchor and focus nodes can be located in a , if supported by the
```

Comment:
Seems like we might not be processing links correctly: https://developer.mozilla.org/en-US/docs/Web/API/Selection/setBaseAndExtent.

Reply:
Yes, likely we don't support this particular construct:

```
... can be located in a {{glossary("shadow tree")}}, if supported by ...
```

Reply:
https://github.com/dart-lang/web/issues/245